.travis.yml

```diff
@@ -4,6 +4,8 @@ matrix:
         -   env: TOXENV=py27
         -   env: TOXENV=py36
             python: 3.6
+        -   env: TOXENV=py37
+            python: 3.7
```

Comment:
this python3.7 is 3.7a4 -- I'm planning to do this in a blanket way once travis-ci has better support

For more information: https://github.com/deadsnakes/travis-ci-python3.7-example#update-2018-07-03